packages/grafana-ui/src/utils/measureText.ts

```diff
@@ -1,6 +1,21 @@
+import tinycolor from 'tinycolor2';
+
 let canvas: HTMLCanvasElement | null = null;
 const cache: Record<string, TextMetrics> = {};
 
+function getCanvasContext() {
+  if (canvas === null) {
+    canvas = document.createElement('canvas');
+  }
+
+  const context = canvas.getContext('2d');
```

Comment:
no cache?

Reply:
reading docs for getContext it looks like this is cached internally in the canvas instance, all subsequent calls to getContext all get the same instance, so do not think there is anything gained by caching this call